### DISCUSSION TITLE
running gnb on a different vm

### DISCUSSION THREAD

**Discussion by abida-02** [srsran/oran-sc-ric#34] [General]
i want to run the gnb with usrp x310 on a separate vm. how can i configure the ric(oran-sc-ric) docker version  in order to integrate with gnb?

**Comment by pgawlowicz** [comment]
please add the following lines to the `docker-compose.yml` file in the `e2term` section:

```
e2term:
    ...
    ports:
      - "36421:36421/sctp"
```

then in the gnb config:

```
e2:
  ...
  addr: x.x.x.x           -> IP address of the PC runing RIC docker container
  bind_addr: y.y.y.y   -> IP address of the gnb PC
  ...
```

**Comment by abida-02** [comment]
Thank you so much. It worked !!
